lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyConstants.java

```diff
@@ -22,9 +24,13 @@ private LibertyConstants() {
 
     public static final String FEATURE_MANAGER_ELEMENT = "featureManager";
     public static final String FEATURE_ELEMENT = "feature";
+    public static final String INCLUDE_ELEMENT = "include";
 
     public static final String PUBLIC_VISIBILITY = "PUBLIC";
 
     public static final String DEFAULT_SERVER_VERSION = "20.0.0.9";
 
+    public static final String WLP_USER_CONFIG_DIR = String.join(File.separator, "usr", "shared", "config");
```

Comment:
Does the resulting String start with or end with a file separator? The way you are using these constants now to determine if a file contains a config dir path, they should really have a file separator at both the beginning and end.